Vagrantfile

```diff
@@ -44,38 +45,26 @@ MASTER_CPUS = cfg["master_cpus"]
 SLAVES_CPUS = cfg["slaves_cpus"]
 PARAVIRT_PROVIDER = cfg.fetch('paravirtprovider', false)
 PREPROVISIONED = cfg.fetch('preprovisioned', true)
-SOLAR_DB_BACKEND = cfg.fetch('solar_db_backend', 'riak')
-
-# Initialize noop plugins only in case of PXE boot
-require_relative 'bootstrap/vagrant_plugins/noop' unless PREPROVISIONED
-
-def ansible_playbook_command(filename, args=[])
-  "ansible-playbook -v -i \"localhost,\" -c local /vagrant/deploy/solar/bootstrap/playbooks/#{filename} #{args.join ' '}"
-end
 
 def shell_script(filename, env=[], args=[])
   "/bin/bash -c \"#{env.join ' '} #{filename} #{args.join ' '} \""
 end
 
-solar_script = ansible_playbook_command("solar.yaml")
-# NOTE(bogdando) w/a for a centos7 issue
-fix_six = shell_script("/vagrant/deploy/solar/bootstrap/playbooks/fix_centos7_six.sh")
+# solar_script = ansible_playbook_command("solar.yaml")
```

Comment:
You deleted ansible_playbook_command function, maybe just delete this line instead of commenting out?